docs/integrations/integration_datahub.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Integrating DataHub With Great Expectations
+authors:
+    name: Mayuri Nehate, John Joyce, Maggie Hays 
+    url: https://datahubproject.io
+---
+
+:::info
+* Maintained By: DataHub
+* Status: Beta
+* Support/Contact: https://slack.datahubproject.io/
+:::
+
+### Introduction
+This integration allows you to push the results of running expectations into DataHub (https://datahubproject.io/). DataHub is a metadata platform which enables search & discovery, federated governance, and data observability for the Modern Data Stack.
+
+
+### Technical background
+There is a custom action named `DataHubValidationAction` which allows you to view expectation results inside of DataHub.
+
+:::note Prerequisites
+ - Create a [Great Expectations Checkpoint](https://docs.greatexpectations.io/docs/terms/checkpoint)
+ - [Deploy an instance of DataHub](https://datahubproject.io/docs/quickstart)
+:::
+
+`DataHubValidationAction` pushes expectations metadata to DataHub. This includes
+
+- **Expectation Details**: Details of assertions (i.e. expectation) set on a Dataset (Table). Expectation set on a dataset in GE aligns with `AssertionInfo` aspect in DataHub. AssertionInfo captures the dataset and dataset fields on which assertion is applied, along with its scope, type and parameters. 
+- **Expectation Results**: Evaluation results for an assertion tracked over time. 
+Validation Result for an expectation in GE align with `AssertionRunEvent` aspect in DataHub. AssertionRunEvent captures the time at which validation was run, batch(subset) of dataset on which it was run, the success status along with other result fields.
+
+
+### Dev loops unlocked by integration
+* View dataset and column level expectations set on a dataset
+* View time-series history of expectation's outcome (pass/fail)
+* View current health status of dataset
+
+### Setup
+
+Install the required dependency in your Great Expectations environment.  
+```shell
+pip install 'acryl-datahub[great-expectations]'
+```
+
+## Usage
+
+:::tip
+Stand up and take a breath
+:::
+
+####  1. Ingest the metadata from source data platform into DataHub
+For example, if you have GE checkpoint that runs expectations on a BigQuery dataset, then first
+ingest the respective dataset into DataHub using [BigQuery](https://datahubproject.io/docs/generated/ingestion/sources/bigquery#module-bigquery) metadata ingestion source recipe. 
+
+```bash
+datahub ingest -c recipe.yaml
+```
+You should be able to see the dataset in DataHub UI.
+
+#### 2. Update GE Checkpoint Configurations
+Add `DataHubValidationAction` in action_list of your Great Expectations Checkpoint. For more details on setting action_list, see [Checkpoints and Actions](https://docs.greatexpectations.io/docs/reference/checkpoints_and_actions/) 
```

Comment:
```suggestion
Add `DataHubValidationAction` in `action_list` of your Great Expectations Checkpoint. For more details on setting action_list, see [the configuration section of the GE Actions reference entry](https://docs.greatexpectations.io/docs/terms/action#configuration) 
```

The previous link that was used here ( https://docs.greatexpectations.io/docs/reference/checkpoints_and_actions/ ) points to a document that is not currently linked to our table of contents and is slated to be removed.  I would suggest using the link to the configuration section of our Actions technical term doc, which I added as part of this suggestion.